src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs

```diff
@@ -54,12 +54,24 @@ public ActionResult Index()
             return View();
         }
 
+        [HttpGet]
+        [SmallProducerSubmissionContext]
+        public ActionResult AlreadySubmittedAndPaid()
+        {
+            return View(SmallProducerSubmissionData.OrganisationData.Id);
+        }
+
         [SmallProducerSubmissionContext]
         [HttpGet]
         public async Task<ActionResult> TaskList()
         {
             var submission = SmallProducerSubmissionData.CurrentSubmission;
 
+            if (submission.HasPaid && submission.Status == SubmissionStatus.Submitted)
```

Comment:
@seanciv  The issue with this is that the user will be able to directly access the pages if they have the URL favourited for example, or just by previous history access. I added some notes into the story about creating an attribute that we can associate with all the actions which would prevent the user being able to "back door" into the screen.